packages/cli/src/util/colorizer.ts

```diff
@@ -0,0 +1,24 @@
+import { mapValues, uniq, isArray } from 'lodash';
+import { Dictionary } from '@stoplight/types';
+
+type ParamValue = number | string | string[] | unknown;
+export type ValuesTransformer = (values: Dictionary<ParamValue>) => Dictionary<string | string[]>;
+
+export const PRE_PARAM_VALUE_TAG = 'PRE_PARAM_VALUE_TAG';
+export const POST_PARAM_VALUE_TAG = 'POST_PARAM_VALUE_TAG';
+
+export function transformPathParamsValues(path: string, transform: (aString: string) => string): string {
+  const taggedParamsValues = new RegExp(`(${PRE_PARAM_VALUE_TAG},?)(.*?)(,?${POST_PARAM_VALUE_TAG})`, 'gm');
```

Comment:
I'd be cautious with omitting those `,`. What if a value is exactly `,,,,,`?

Reply:
I'll add some tests for this use case, thanks!